learning/step_by_step/scripting.rst

```diff
@@ -41,8 +41,8 @@ available engine classes.
 GDScript
 --------
 
-:ref:`doc_gdscript` is a dynamically typed scripting language to fit
-inside Godot. It was designed with the following goals:
+:ref:`doc_gdscript` is a dynamically-typed scripting language designed to
```

Comment:
"dynamically-typed" does not have a hyphen:

https://www.google.com/search?q=dynamically+typed